.github/workflows/lint-build.yml

```diff
@@ -3,8 +3,14 @@ name: Lint & Build
 on:
   workflow_dispatch:
   pull_request:
+    paths:
+      - "**/*.user.css"
+      - "**/*.ts"
   push:
     branches: [main]
+    paths:
+      - "**/*.user.css"
+      - "**/*.ts"
```

Comment:
When the `userstyles.yml` file changes this should run as well? IIRC it lints that too?